### ISSUE TITLE
Question about Distributed training

### ISSUE THREAD

**Issue by jiaruHithub** [rusty1s/pytorch_geometric#1704]
## ❓ Questions & Help

I want to know whether PyG support the Distributed training.
And is the Distributed training usage about PyG just like else?

**Comment by rusty1s** [comment]
This is on the [roadmap for PyG 1.7.0](https://github.com/rusty1s/pytorch_geometric/issues/1465), and there is an open PR to demonstrate, see https://github.com/rusty1s/pytorch_geometric/pull/1453.

**Comment by jiaruHithub** [comment]
Fine 3Q for your working.
And I have a question for multi-GPUS
Do you think this will improve the performance of models in graph classification field?

**Comment by rusty1s** [comment]
Do you mean model performance or training time? IMO, distributed training/data parallel training is all about larger batch sizes to obtain more accurate gradients. This can but does not need to increase model performance.

**Comment by jiaruHithub** [comment]
> Do you mean model performance or training time? IMO, distributed training/data parallel training is all about larger batch sizes to obtain more accurate gradients. This can but does not need to increase model performance.

Ok,your mean is distributed training trackle the issue of memory caused by larger batchsize but cannot increase the acc of one model.
